web_drop_target/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+Drop target support
+===================
+
+This module was written to extend the functionality of the web client to support dropping local files into the web client.
+
+By default, this will create an attachment when a file is being dropped on a form, other modules can add more features.
```

Comment:
By default, an attachment will be created when dropping a file on a form.
Further, this module is meant as a base drag&drop module supporting other actions after some file is dropped so that other modules can add more features.